zerver/tests/test_home.py

```diff
@@ -393,7 +393,7 @@ def test_num_queries_for_realm_admin(self) -> None:
         # Verify number of queries for Realm admin isn't much higher than for normal users.
         self.login("iago")
         flush_per_request_caches()
-        with self.assert_database_query_count(45):
+        with self.assert_database_query_count(46):
```

Comment:
I think you should be able tell that your changes creates an increase in queries and not a reduction. This change was unnecessary and hence the failing testcases.

Reply:
I understand that my changes do not create a reduction. 
The change that you have mentioned was unnecessary, I made because I was failing a previous test case keeping this 45. I am not sure if I was correct to do so.

Reply:
okay i think i got it, passed the test case